lang/cs/Org.Apache.REEF.Tang/Formats/AvroConfigurationSerializer.cs

```diff
@@ -260,6 +260,28 @@ public AvroConfiguration ToAvroConfiguration(IConfiguration c)
                 l.Add(new ConfigurationEntry(e.Key.GetFullName(), val));
             }
 
+            IEnumerator bl = conf.GetBoundList().GetEnumerator();
+            while (bl.MoveNext())
+            {
+                KeyValuePair<INamedParameterNode, IList<object>> e = (KeyValuePair<INamedParameterNode, IList<object>>)bl.Current;
+                foreach (var item in e.Value)
+                {
+                    string val = null;
+                    if (item is string)
+                    {
+                        val = (string)item;
+                    }
+                    else if (item is INode)
+                    {
+                        val = ((INode)item).GetFullName();
+                    }
+                    else
+                    {
+                        Org.Apache.REEF.Utilities.Diagnostics.Exceptions.Throw(new IllegalStateException(), LOGGER);
```

Comment:
please don't use the `Exceptions` class anymore. Instead, just `throw` the exception.